Makefile.common

```diff
@@ -238,8 +238,12 @@ endif
 
 .PHONY: common-build
 common-build: promu
+	@echo ">> compressing assets"
+	@scripts/compress_assets.sh
 	@echo ">> building binaries"
 	GO111MODULE=$(GO111MODULE) $(PROMU) build --prefix $(PREFIX) $(PROMU_BINARIES)
```

Comment:
If anything goes wrong in this step, the assets stay compressed... could we decompress them not in-place, but to some alternative temporary place, and then embed them from there?

Reply:
`embed` will only accept directives that are below or at the same level as the including file (directive may not start the `/` or include `..`) so anything like that would have to happen in `web/ui/`. I think this requirement will mess with builds not built with `builtinassets`.
I figured the assets staying compressed is the least bad solution as git would easily be able to revert that.

Reply:
@juliusv How about this solution: the three steps are executed in a single sub-shell, i.e. if the build has an error, the decompress step is run regardless.

Reply:
I added the cleanup step as a trap on EXIT. This should now always decompress the assets whether the build fails or succeeds, while returning the correct exit code.
@juliusv I'd much appreciate if you could take another look.